config/jobs/kubernetes/test-infra/janitors.yaml

```diff
@@ -82,15 +82,15 @@ periodics:
   name: maintenance-pull-janitor
   labels:
     preset-service-account: "true"
+  decorate: true
+  decoration_config:
+    timeout: 30m
   spec:
     containers:
-    - args:
-      - --bare
-      - --timeout=30
-      - --service-account=/etc/service-account/service-account.json
```

Comment:
this flag for the service account  👀 , @BenTheElder @ameukam

Reply:
automatically injected

Reply:
the janitor job scares me if something goes wrong, will defer to others more familiars

Reply:
we're gonna try it out and file a revert if needed